README.md

```diff
@@ -51,6 +61,29 @@ You most likely may override `image`, `applicationPort` and `environment` for yo
 | `livenessTimeout` | Liveness probe timeout (seconds) | `3` |
 | `livenessPeriod` | Liveness probe period (seconds) | `15` |
 | `livenessFailureThreshold`| Liveness failure threshold | `3` |
+| `keyVaults`| This section is about adding the keyvault secrets to the file system see [Adding Azure Key Vault Secrets]()| none |
```

Comment:
how about matching it to the other chart
```suggestion
| `keyVaults`| Mappings of keyvaults to be mounted as flexvolumes (see Example Configuration)| none |
```

Reply:
Im referencing the section where I've actually spelt out the information. 
Since its unlike the other sections a slightly less techy sentence might be easier to read ?